CHANGES.rst

```diff
@@ -1,6 +1,17 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds GitLab projects integration.
+- Adds CERN SSO integration and login/logout handling.
+- Adds state management with Redux.
+- Improves UX in projects page.
+- Allows enabling/disabling GitLab project integration.
+- Includes SASS and CSS-modules support.
```

Comment:
Can you group items roughly as follows:

- first basic login/user page i.e. SSO login plus "temporary token page"  which could be better said "simple user page showing access token" for example 
- then all GitLab ones together related to integration and project management
- end with all technology items, i.e. sass and css modules